src/Components/OATConfirmDialog/OATConfirmDialog.tsx

```diff
@@ -1,11 +1,12 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import { useOatPageContext } from '../../Models/Context/OatPageContext/OatPageContext';
 import ConfirmDeleteDialog from '../ADT3DSceneBuilder/Internal/ConfirmDeleteDialog/ConfirmDeleteDialog';
 import { OatPageContextActionType } from '../../Models/Context/OatPageContext/OatPageContext.types';
 
 const OATConfirmDialog: React.FC = () => {
```

Comment:
If this confirm dialog only for deletion, we can also change the name of the component maybe to "OATConfirmDeletionDialog"?

Reply:
yeah. it was originally that way, but wont' be going forward.